routes/bridges/gassistant/index.js

```diff
@@ -0,0 +1,118 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of ThingEngine
+//
+// Copyright 2017-2019 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Swee Kiat Lim <sweekiat@cs.stanford.edu>
+//
+// See COPYING for details
+"use strict";
+
+const Q = require('q');
+const express = require('express');
+
+const EngineManager = require('../../../almond/enginemanagerclient');
+const { actionssdk, Image } = require('actions-on-google');
+
+const userUtils = require('../../../util/user');
+
+var router = express.Router();
+
+class GoogleAssistantDelegate {
+    constructor(locale) {
+        this._buffer = '';
+        this._locale = locale;
+    }
+
+    send(text, icon) {
+        this._buffer += text + '\n';
+    }
+
+    sendPicture(url, icon) {
+        this._image = new Image({
+            url: url,
+            alt: url,
+        });
+    }
+
+    sendRDL(rdl, icon) {
+        this._buffer += rdl.displayTitle + '\n';
+    }
+
+    sendChoice(idx, what, title, text) {
+        this._buffer += title + '\n';
+    }
+
+    sendButton(title, json) {
+        // FIXME
+    }
+
+    sendLink(title, url) {
+        if (url === '/user/register') {
+            this._card = {
+                type: 'LinkAccount'
+            };
```

Comment:
Is this right or is it a copy-paste from Alexa?

Reply:
Sorry! Removed this for now